src/bootstrap/defaults/config.library.toml

```diff
@@ -1,9 +1,9 @@
 # These defaults are meant for contributors to the standard library and documentation.
 [build]
 # When building the standard library, you almost never want to build the compiler itself.
```

Comment:
nit: this is no longer true, right? you always *have to* build the compiler.

Reply:
download-rustc is a thing. In fact if I recall correctly it was introduced exactly to avoid having to build rustc yourself when only making a library change.

Reply:
Sure, but this comment is still not coherent with `stage = 1`s. I guess that could be moved to the `download-rustc` line?